src/main/java/frc/robot/Robot.java

```diff
@@ -64,12 +66,13 @@ public void simulationPeriodic() {}
 
   // Controller Configuration
   public void configureTeleopControls() {
-    if (controller.getAButton()) {
-      gutPrototype.spin(0.8);
+    // Intake
+    if (controller.getRightTriggerAxis() >= 0.0) {
```

Comment:
Here, you want to add a dead band. Otherwise, the intake might spin at a very slow rate, even when you're not pressing the button, because it's registering some stupid-small value from the trigger.

Reply:
Could you explain how I'd go about doing that? Would I just check if the value is between a certain threshold (maybe -0.1 to 0.1) and set the value to 0 then?